alert-policies/amazon-workmail/IncomingEmailBounced.yml

```diff
@@ -0,0 +1,27 @@
+name: High IncomingEmailBounced
+
+description: |+
+  This alert is triggered when the IncomingEmailBounced is above 100 in 10 minutes.
```

Comment:
```suggestion
  This alert is triggered when the Incoming Email Bounced is above 100 in 10 minutes.
```

Reply:
@josephgregoryii , Addressed the changes